website/pages/en/network/indexing.mdx

```diff
@@ -662,21 +662,21 @@ ActionType {
 Example usage from source:
 
 ```bash
-indexer indexer actions get all
+graph indexer indexer actions get all
```

Comment:
Is the double `indexer` intentional?

Reply:
Wow, haha no, typo on my part. Thanks for flagging!